src/CheckoutSdk/Disputes/DisputesClient.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.Disputes
+{
+    /// <summary>
+    /// Default implementation of <see cref="IDisputesClient"/>.
+    /// </summary>
+    public class DisputesClient : IDisputesClient
+    {
+        private readonly IApiClient _apiClient;
+        private readonly IApiCredentials _credentials;
+
+        public DisputesClient(IApiClient apiClient, CheckoutConfiguration configuration)
+        {
+            _apiClient = apiClient ?? throw new ArgumentNullException(nameof(apiClient));
+            if (configuration == null) throw new ArgumentNullException(nameof(configuration));
+
+            _credentials = new SecretKeyCredentials(configuration);
+        }
+
+        public Task<GetDisputesResponse> GetDisputesAsync(
+            int? limit = null,
+            int? skip = null,
+            string from = "",
+            string to = "",
+            string id = "",
+            string statuses = "",
+            string paymentId = "",
+            string paymentReference = "",
+            string paymentArn = "",
+            bool? thisChannelOnly = null,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            string path = "disputes?";
+            if (limit.HasValue) path += $"limit={limit}";
```

Comment:
As a minimum, you'd be better off using a `StringBuilder` here as it avoids allocating a new string for every change.

However, you might want to consider handling this in a cleaner way similar to what I did (in a past life) [here](https://github.com/FabrikCreations/fabrikapi-client-net/blob/master/src/Fabrik.API.Client.Core/ApiClient.cs#L304).

```
public Task<GetDisputesResponse> GetDisputesAsync(GetDisputesRequest request, CancellationToken cancellationToken)
{
    string pathWithQuery = request.BuildPath(path);
    // etc.
}
```

Ideally the API would then be something like:

Reply:
Would you rather have `GetDisputesRequest` object handle the build of the string, or should the `DisputesClient` do it similar to `PaymentsClient` here https://github.com/checkout/checkout-sdk-net/blob/develop/src/CheckoutSdk/Payments/PaymentsClient.cs#L73?